opacclient/opacapp/src/main/java/de/geeksfactory/opacclient/storage/AccountDataSource.java

```diff
@@ -284,6 +288,8 @@ private void setAccountItemAttributes(AccountItem item, ContentValues cv, long a
         putOrNull(cv, "format", item.getFormat());
         putOrNull(cv, "itemid", item.getId());
         putOrNull(cv, "status", item.getStatus());
+        putOrNull(cv, "cover", item.getCover());
+        putOrNull(cv, "mediatype", item.getMediaType().toString());
```

Comment:
This crashed for me repeatedly on APIs where getMediaType returns null, e.g. the Test API or Adis.